modules.json

```diff
@@ -0,0 +1,284 @@
+[
+  {
+    "repo_name": "cisco-network-puppet-module",
+    "github_namespace": "cisco",
+    "module_name": "network-puppet-module",
+    "forge_name": "puppetlabs-ciscopuppet"
+  },
+  {
+    "repo_name": "puppetlabs-accounts",
+    "github_namespace": "puppetlabs",
+    "module_name": "accounts",
+    "forge_name": "puppetlabs-accounts"
```

Comment:
there is only one use of `forge_name` in the code today, but mushing together the forge namespace and the module name into one values makes it a lot harder to use the data elsewhere.